election-server/src/bin/http.rs

```diff
@@ -102,6 +126,31 @@ struct AppState {
     db_pool: DatabasePool,
 }
 
+#[derive(Clone, Debug, clap::ValueEnum, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+enum Network {
+    Mainnet,
+    Testnet,
+}
```

Comment:
Maybe just use the one defined in `web3id` in the rust SDK instead of duplicating it.